apps/damap-frontend/src/app/components/landing-page/landing-page.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="background">
+  <div class="container">
+    <!--
+    For now commented out, until we have some info we can display here
+    <div class="info-bar fadeInTop">
+      <div class="info-bar-typography"> Usage 29 | DMPs 311 | Related Organizations 322</div>
+    </div>
+    -->
+    <div class="double-items">
+      <div class="left-side-container">
+        <div class="title-typography fadeInTop">
+          DAMAP - a tool for machine actionable DMPs
```

Comment:
please make sure to use translatable strings in the UI. Allows for internationalization as well as easy configuration of the text, if it does not fit.

Reply:
Yes good point, i glossed over that completely. I added the translatable strings now.